frontend/study/Study.js

```diff
@@ -14,14 +14,15 @@ import {renderStudyDisplay} from "riskofbias/robTable/components/StudyDisplay";
 import {SCORE_SHADES, SCORE_TEXT} from "riskofbias/constants";
 
 class Study {
-    constructor(data) {
```

Comment:
some of the stuff in Study.js will need to be reverted w/ my recent changes; i noticed in a few other places as well...